features/step_definitions/basic_steps.rb

```diff
@@ -46,3 +37,18 @@ def page_path_from(page_name)
 Then("I should not see {string}") do |content|
   expect(page).not_to have_content content
 end
+
+def page_path_from(page_name)
+  case page_name.downcase
+    when 'thaitanic' then
```

Comment:
If the action is on the next line the `then` isn't required. Also you need a default case. What happens when we don't match any of the pre-defined page names?